mmtrack/models/trackers/strongsort_tracker.py

```diff
@@ -0,0 +1,251 @@
+
+    Args:
+        obj_score_thr (float, optional): Threshold to filter the objects.
+            Defaults to 0.6.
+        reid (dict, optional): Configuration for the ReID model.
+            - num_samples (int, optional): Number of samples to calculate the
+                feature embeddings of a track. Default to None.
+            - image_scale (tuple, optional): Input scale of the ReID model.
+                Default to (256, 128).
+            - img_norm_cfg (dict, optional): Configuration to normalize the
+                input. Default to None.
+            - match_score_thr (float, optional): Similarity threshold for the
+                matching process. Default to 0.3.
+            - motion_weight (float, optional): the weight of the motion cost.
+                Defaults to 0.02.
+        match_iou_thr (float, optional): Threshold of the IoU matching process.
+            Defaults to 0.7.
+        num_tentatives (int, optional): Number of continuous frames to confirm
+            a track. Defaults to 2.
+    """
+
+    def __init__(self,
+                 obj_score_thr: float = 0.6,
+                 reid: dict = dict(
+                     num_samples=None,
+                     img_scale=(256, 128),
+                     img_norm_cfg=None,
+                     match_score_thr=0.3,
+                     motion_weight=0.02),
+                 match_iou_thr: float = 0.7,
+                 num_tentatives: int = 2,
+                 **kwargs):
+        super().__init__(**kwargs)
+        self.obj_score_thr = obj_score_thr
+        self.reid = reid
+        self.match_iou_thr = match_iou_thr
+        self.num_tentatives = num_tentatives
+
+    def update_track(self, id: int, obj: Tuple[Tensor]) -> None:
+        """Update a track."""
+        for k, v in zip(self.memo_items, obj):
+            v = v[None]
+            if self.momentums is not None and k in self.momentums:
+                m = self.momentums[k]
+                self.tracks[id][k] = (1 - m) * self.tracks[id][k] + m * v
+            else:
+                self.tracks[id][k].append(v)
+
+        if self.tracks[id].tentative:
+            if len(self.tracks[id]['bboxes']) >= self.num_tentatives:
+                self.tracks[id].tentative = False
+        bbox = bbox_xyxy_to_cxcyah(self.tracks[id].bboxes[-1])  # size = (1, 4)
+        assert bbox.ndim == 2 and bbox.shape[0] == 1
+        bbox = bbox.squeeze(0).cpu().numpy()
+        score = float(self.tracks[id].scores[-1].cpu())
+        self.tracks[id].mean, self.tracks[id].covariance = self.kf.update(
+            self.tracks[id].mean, self.tracks[id].covariance, bbox, score)
+
+    def track(self,
+              model: torch.nn.Module,
+              img: Tensor,
+              feats: List[Tensor],
+              data_sample: TrackDataSample,
+              data_preprocessor: OptConfigType = None,
+              rescale: bool = False,
+              **kwargs) -> InstanceData:
+        """Tracking forward function.
```

Comment:
Does the only difference between `update_track`, `track` functions in `StrongSORTTracker` and `update_track`, `track` functions in `DeepSORTTracker` be that we pass `score` into `self.kf.update()`

Reply:
yes for `update_track`.

but for `track`, there are also some differences in reid process, motion process, reid dist and motion dist. 
For example, the motion dist is added to the matching cost in StrongSORT, but not in DeepSORT.